frontend/www/js/omegaup/components/arena/Runs.vue

```diff
@@ -620,18 +764,87 @@ export default class Runs extends Vue {
 
   runtime(run: types.Run): string {
     if (
-      run.status == 'ready' &&
-      run.verdict != 'JE' &&
-      run.verdict != 'VE' &&
-      run.verdict != 'CE'
+      run.status !== 'ready' ||
+      run.status_runtime === RuntimeStatus.NotAvailable
     ) {
-      let prefix = '';
-      if (run.verdict == 'TLE') {
-        prefix = '>';
-      }
-      return `${prefix}${(run.runtime / 1000).toFixed(2)} s`;
+      return '—';
+    }
+
+    if (run.status_runtime === RuntimeStatus.Exceeded) {
+      return T.runDetailsExceeded;
+    }
+
+    return `${(run.runtime / 1000).toFixed(2)} s`;
+  }
+
+  execution(run: types.Run): string {
+    if (run.status !== 'ready') {
+      return '—';
+    }
+
+    switch (run.execution) {
+      case ExecutionStatus.JudgeError:
+        return T.runDetailsJudgeError;
+      case ExecutionStatus.ValidatorError:
+        return T.runDetailsValidatorError;
+      case ExecutionStatus.CompilationError:
+        return T.runDetailsCompilationError;
+      case ExecutionStatus.RuntimeFunctionError:
+        return T.runDetailsRuntimeFunctionError;
+      case ExecutionStatus.RuntimeError:
+        return T.runDetailsRuntimeError;
+      case ExecutionStatus.Interrupted:
+        return T.runDetailsInterrupted;
+      default:
+        return T.runDetailsFinished;
+    }
+  }
+
+  output(run: types.Run): string {
+    if (run.status !== 'ready') {
+      return '—';
+    }
+
+    switch (run.output) {
+      case StringOutputStatus.Exceeded:
+        return T.runDetailsExceeded;
+      case StringOutputStatus.Incorrect:
+        return T.runDetailsIncorrect;
+      case StringOutputStatus.Interrupted:
+        return T.runDetailsInterrupted;
+      default:
+        return T.runDetailsCorrect;
+    }
+  }
+
+  statusPercentageClass(run: types.Run): string {
+    if (run.status !== 'ready') {
+      return '';
+    }
+
+    if (run.type == 'disqualified') {
+      return 'status-disqualified';
+    }
+
+    return 'status-ac';
+  }
+
+  outputIconColorStatus(run: types.Run): number {
+    if (
+      !(
+        run.status === 'ready' &&
+        run.output !== StringOutputStatus.Exceeded &&
+        run.output !== StringOutputStatus.Interrupted
+      )
+    ) {
+      return NumericOutputStatus.None;
+    }
+
+    if (run.output !== StringOutputStatus.Incorrect) {
+      return NumericOutputStatus.Correct;
+    } else {
+      return NumericOutputStatus.Incorrect;
```

Comment:
Guard clause pattern:
```suggestion
    if (run.output !== StringOutputStatus.Incorrect) {
      return NumericOutputStatus.Correct;
    }
    return NumericOutputStatus.Incorrect;
```

Reply:
Listo, solo que mis últimos cambios no están pasando una prueba de Cypress, más tarde la reviso